ArduPlane/RC_Channel.cpp

```diff
@@ -86,3 +86,23 @@ void RC_Channel_Plane::do_aux_function(const aux_func_t ch_option, const aux_swi
         break;
     }
 }
+
+void RC_Channel_Plane::do_aux_function_emergency_stop(const aux_switch_pos_t ch_flag)
```

Comment:
This is the same across all vehicles; we should have a pure-virtual `set_emergency_stop` function implemented by all the vehicles.